src/dosarray_graphing_focus.sh

```diff
@@ -5,13 +5,26 @@
 #
 # Use of this source code is governed by the Apache 2.0 license; see LICENSE
 #
-# $ DOSARRAY_HIST_FOCUS=10 python ../generate_availability_chart.py '../../exp4/sl_60total_start10_last20_apache_event/c*.log' > availability.data
-# $ ../dosarray_graphing_focus.sh . '' 10 30
+# $ export DOSARRAY_VERSION="0.3"
+# $ DOSARRAY_NHIST_RESULT=1 python generate_availability_chart.py '../example_experiment/c*log' > availability_focus.data
```

Comment:
Don't we need to set `DOSARRAY_HIST_FOCUS` to get the desired output, as shown in the comment of the earlier revision of this file?

Reply:
Thats a mistake on my part. Will fix it in the next commits